samples/snippets/fsharp/tour.fs

```diff
@@ -457,7 +457,7 @@ module Sequences =
 ///
 /// Recursion is the preferred way to process sequences or collections in F#.
 ///
-/// To learn more, see: https://docs.microsoft.com/dotnet/articles/fsharp/language-reference/functions/index#recursive-functions
+/// To learn more, see: https://docs.microsoft.com/dotnet/fsharp/language-reference/functions/index#recursive-functions
```

Comment:
... and here ...

*https://docs.microsoft.com/dotnet/fsharp/language-reference/functions/#recursive-functions*